Control-User-Interfaces/writing-json-block-files.rst

```diff
@@ -0,0 +1,506 @@
+                }
+            },
+            {
+                "name": "auxtel/offset_ataos.py",
+                "standard": true,
+                "parameters": {
+                    "z": 0.025
+                }
+            },
+            {
+                "name": "auxtel/take_image_latiss.py",
+                "standard": true,
+                "parameters": {
+                    "program": "BLOCK-91",
+                    "reason": "SITCOM-1012",
+                    "nimages": 2,
+                    "exp_times": 10,
+                    "image_type": "FOCUS",
+                    "filter": "cyl_lens",
+                    "grating": "holo4_003"
+                }
+            },
+            {
+                "name": "auxtel/offset_ataos.py",
+                "standard": true,
+                "parameters": {
+                    "z": 0.025
+                }
+            },
+            {
+                "name": "auxtel/take_image_latiss.py",
+                "standard": true,
+                "parameters": {
+                    "program": "BLOCK-91",
+                    "reason": "SITCOM-1012",
+                    "nimages": 2,
+                    "exp_times": 10,
+                    "image_type": "FOCUS",
+                    "filter": "cyl_lens",
+                    "grating": "holo4_003"
+                }
+            },
+            {
+                "name": "auxtel/offset_ataos.py",
+                "standard": true,
+                "parameters": {
+                    "z": 0.025
+                }
+            },
+            {
+                "name": "auxtel/take_image_latiss.py",
+                "standard": true,
+                "parameters": {
+                    "program": "BLOCK-91",
+                    "reason": "SITCOM-1012",
+                    "nimages": 2,
+                    "exp_times": 10,
+                    "image_type": "FOCUS",
+                    "filter": "cyl_lens",
+                    "grating": "holo4_003"
+                }
+            },
+            {
+                "name": "auxtel/offset_ataos.py",
+                "standard": true,
+                "parameters": {
+                    "z": -0.1
+                }
+            },
+            {
+                "name": "auxtel/stop_tracking.py",
+                "standard": true,
+                "parameters": {}
+            }
+        ]
+    }
+
+.. _Writing-JSON-BLOCK-Files-Notebook:
+
+Generating this BLOCK in a Notebook
+-----------------------------------
+
+To generate this JSON BLOCK, we will make use of a set of convenience classes defined in the `ts observing`_ repository. 
+The `ts observing`_ repository is also used by the Scheduler CSC when loading JSON BLOCKS, 
+so by using it to generate our JSON file we can ensure proper formatting. 
+
+.. _ts observing: https://github.com/lsst-ts/ts_observing
+
+The `ts observing`_ repository can be installed in your local environment by running
+
+.. prompt:: bash
+    
```

Comment:
I think the best way to install this package is actually to use conda. This packages has some dependencies that are not pulled when installed this way. This is how one would install the package for development. 

Also, we should probably add this to the nublado images. 

I would recommend replacing it with something like this:

```
conda install -c lsstts -y ts-observing
```